package-managers/qubes-download-dom0-updates.sh

```diff
@@ -166,14 +158,9 @@ mkdir -p "$DOM0_UPDATES_DIR/packages"
 set -e
 
 if [ "$GUI" = 1 ]; then
-    ( echo "1"
-    # shellcheck disable=SC2086
-    $YUM_COMMAND $OPTS "${PKGLIST[@]}"
-    echo 100 ) | zenity --progress --pulsate --auto-close --auto-kill \
-         --text="Downloading updates for Dom0, please wait..." --title="Qubes Dom0 updates"
+    xterm -e sh -c '"$0" "$@"; read' "${YUM_COMMAND[@]}" "${OPTS[@]}" "${PKGLIST[@]}"
```

Comment:
shellcheck (incorrectly) complains about this line, add a comment to silent this one